src/commands/logout/functions.ts

```diff
@@ -17,12 +18,24 @@ export default class Login extends Command {
 
   static examples = messages.getMessages('examples');
 
+  static flags = {
+    json: FunctionsFlagBuilder.json,
+  };
+
   async run() {
+    const { flags } = await this.parse(Login);
     cli.action.start(messages.getMessage('action.start'));
 
     this.globalInfo.tokens.unset(Command.TOKEN_BEARER_KEY);
     await this.globalInfo.write();
 
     cli.action.stop();
+    if (flags.json) {
+      cli.styledJSON({
+        status: 0,
+        result: [],
```

Comment:
No that we have this standardized, but I'd make the result a string that says "Logged out." or whatnot.

Reply:
In the PRD, Shyam has it like this? Want to chat with him on it tomorrow?